### ISSUE TITLE
[Backend] uwsgi/gunicorn or nginx setup for hug API

### ISSUE THREAD

**Issue by titipata** [titipata/penn-events-calendar#96]
Create a script for setting up uwsgi with gunicorn or nginx. Not sure if [this](https://www.digitalocean.com/community/tutorials/how-to-deploy-python-wsgi-applications-using-uwsgi-web-server-with-nginx) is the good one to follow. 

@kittinan do you know where is a good site to follow for setting up uwsgi/gunicorn for Python backend?

**Comment by titipata** [comment]
To serve `uwsgi` from our hug API, simply run

```sh
uwsgi --http 0.0.0.0:8888 --wsgi-file hug_api.py --callable __hug_wsgi__
```

make sure that `uwsgi` is installed via ` conda install -c conda-forge uwsgi`

**Comment by titipata** [comment]
Now, we want to see if we can change the code in [supervisord configuration file](https://github.com/titipata/penn-events-calendar/blob/master/devops/penn-calendar.supervisor.conf#L186) to use `uwsgi` instead.

**Comment by bluenex** [comment]
We will need Nginx for reverse proxy in the future. Closing this issue for now as it is implemented in #100
